src/schema/v2/me/bidder_position.ts

```diff
@@ -39,7 +39,11 @@ export const BidderPosition: GraphQLFieldConfig<void, ResolverContext> = {
             position.errors &&
             position.errors.includes("Lot must be open")
           ) {
-            status = "SALE_CLOSED"
+            if (position.bidder.sale.auction_state === "open") {
```

Comment:
Should you be checking `auction_state` here rather than the state of an individual sale artwork/lot?

Reply:
Yes. The case we are interested in is that the lot is closed (which is what happens when `position.errors` includes `"Lot must be open"` and the sale is still open.

Reply:
Yea, I think that's right.